src/components/accessories/billTable/RenderBillDetails.tsx

```diff
@@ -0,0 +1,235 @@
+import {
+  Card,
+  CardContent,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@material-ui/core";
+import React, { FC } from "react";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import { FullBillDTO } from "../../../generated";
+import { renderDate } from "../../../libraries/formatUtils/dataFormatting";
+import "./styles.scss";
+
+interface IBillProps {
+  fullBill: FullBillDTO;
+}
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    overflow: "hide",
+    border: "none",
+    boxShadow: "none",
+  },
+  table: {
+    minWidth: 340,
+    padding: 10,
+    whiteSpace: "nowrap",
+  },
+  tableCell: {
+    paddingRight: 4,
+    paddingLeft: 5,
+  },
+}));
+
+const currencyFormat = (num: number | undefined) => {
+  return num
+    ? "$" + num.toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,")
+    : "";
+};
```

Comment:
Few things:

- Put this under /libraries/formatUtils.ts
- this function should accept number and string
- use Numbro (https://numbrojs.com/) in order to accept different currency format (now it is dollar but maybe could be something else in future)

Reply:
OK